### PR TITLE
Update documentation according to the latest functions-framework

### PR DESCRIPTION
**Pull request by tpiperatgod**
Signed-off-by: laminar <fangtian@kubesphere.io>

### CODE REVIEW

File: functions-framework/README.md [hunk 1]
```diff
@@ -4,15 +4,83 @@ This directory holds the validation demos for the [functions-framework](https://
...
+func (p *PluginCustom) ExecPostHook(ctx ofctx.Context, plugins map[string]plugin.Plugin) error {
+	return nil
+}
+```
+
+plugin-example
+
+```go
+func (p *PluginExample) ExecPreHook(ctx ofctx.Context, plugins map[string]plugin.Plugin) error {
+	r := preHookLogic(ctx.Ctx)
```
Comment (benjaminhuo) [inline functions-framework/README.md:52]: where to find code for preHookLogic?

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: functions-framework/README.md (hunk 1 above)
Comment (tpiperatgod) [reply functions-framework/README.md:52]: add the `preHookLogic()`

File: functions-framework/golang/Async/README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# OpenFuncAsync Runtime examples
+# Async Runtime examples
 
 In order to follow these examples, you need to install [Dapr](https://docs.dapr.io/getting-started/install-dapr-selfhost/) and [Go 1.15+](https://golang.org/doc/install).
 
-## OpenFunction Context Function
+## Context signature function
```
Comment (benjaminhuo) [inline functions-framework/golang/Async/README.md:5]: Context signature function => OpenFunction Function

File: functions-framework/golang/Knative/README.md [hunk 3]
```diff
@@ -2,8 +2,13 @@
 
 In order to follow these examples, you need to install [Knative](https://knative.dev/docs/install/) and [Go 1.15+](https://golang.org/doc/install).
 
-## OpenFunction Context Function
+## Http signature function
```
Comment (benjaminhuo) [inline functions-framework/golang/Knative/README.md:5]: Http signature function
=>
Http Function

File: functions-framework/golang/Knative/README.md [hunk 4]
```diff
@@ -2,8 +2,13 @@
...
 
 - HTTP request
-    - [example](simple-hello)
+    - [example](http)
+
+## CloudEvents signature function
```
Comment (benjaminhuo) [inline functions-framework/golang/Knative/README.md:10]: CloudEvents signature function
=>
CloudEvents Function

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**